CHANGELOG.md

```diff
@@ -277,7 +277,7 @@
 ## Version 1.1.5 (15FEB22)
 
 * **Monitoring**
-  * [CHANGE] - The SAS Viya Welcome dashboard has been updated with a
+  * [CHANGE] - The SAS Viya platform Welcome dashboard has been updated with a
```

Comment:
The name of this dashboard (as shown in the list of Grafana dashboards and on the dashboard itself) is "SAS Viya Welcome".  If we need to revisit/revise that, we can discuss.  But, unless it changes, I think the reference here should match what the user will see in Grafana.